srcalc/src/main/java/gov/va/med/srcalc/domain/Patient.java

```diff
@@ -160,6 +164,19 @@ public void setLabs(final Map<String, RetrievedValue> labs)
         fLabs = labs;
     }
     
+    /**
+     * Returns the patient's health factors in String form.
+     */
+    public List<String> getHealthFactors()
+    {
+        return fHealthFactors;
+    }
+    
+    public void setHealthFactors(final List<String> healthFactors)
```

Comment:
Same comment as usual: providing a mutable List and a setter is an unclear interface.